app/services/idv/agent.rb

```diff
@@ -9,7 +9,9 @@ def proof_resolution(
       should_proof_state_id:,
       trace_id:,
       user_id:,
-      threatmetrix_session_id:
+      threatmetrix_session_id:,
+      request_ip:,
+      uuid_prefix:
```

Comment:
ok so same request again, we need to make sure this is plumbed all the way to the user, now that we have added required kewyards for this method, can we update the places that call it?

```
> git grep "\.proof_resolution" -- app 
app/controllers/idv/gpo_controller.rb:      Idv::Agent.new(applicant).proof_resolution(
app/services/idv/steps/verify_base_step.rb:        idv_agent.proof_resolution(
```

Reply:
Two questions:
1- Not seeing proof_resolution in gpo_controller. What line number is yours on?
2- for verify_base_step, what should we set the request_ip to?

Reply:
1. ah I was behind, that's gone now thanks to #6775
2. that's a flow machine step, so it has `@flow` which should have a request, which should have a request IP, but that's a guess I'd recommend double checking 

https://github.com/18F/identity-idp/blob/781377da55a73b9e863acec57bc148398e548abe/app/services/idv/steps/verify_base_step.rb#L162


https://github.com/18F/identity-idp/blob/781377da55a73b9e863acec57bc148398e548abe/app/services/flow/base_flow.rb#L4

Reply:
Looks like a bunch of other tests are failing, maybe missing some others. Going to look at that as well.